docs/source/formulations_example.rst

```diff
@@ -0,0 +1,686 @@
+.. formulations_example:
+
+Formulations Example
+====================
+
+The Citrine Platform is a powerful tool to analyze formulations data.
+A formulation is, generally speaking, a mixture of mixtures.
+Individual ingredients are mixed together to form more complex ingredients, which may themselves be mixed with other ingredients and/or mixtures, and so on.
+Generally, the top-level formulation has some properties that we wish to optimize.
+The ingredients themselves may also have known properties that are useful for understanding the properties of the top-level formulation.
+In addition, each mixing process may be characterized by attributes such as the temperature of a reaction vessel or the speed of a mixing element.
+
+The GEMD data model provides a way to record all of the information relevant to a formulations problem,
+and Citrine's AI Engine turns this information into trained models that can predict the properties of novel formulations.
+This section demonstrates how to express a complete formulations example in Citrine-python, from data ingestion to new candidate generation.
+
+Example Raw Data
+----------------
+
+To demonstrate the formulations machinery, we consider a toy problem -- margaritas!
+We scour our pantry and gather some materials, laid out in the table below.
+These are known as "atomic" materials because for our purposes they are not broken into more fundamental materials.
```

Comment:
I don't think we use any term for this concept anywhere else. The docstring for `SimpleMixturePredictor` just says "flattens to a simple mixture" without explaining what makes it simple.

I like "atomic" over "raw", since the primary quality is that we are uninterested in any smaller components.